src/registrar/tests/test_emails.py

```diff
@@ -74,9 +74,9 @@ def test_submission_confirmation_current_website_spacing(self):
                 application.submit()
         _, kwargs = self.mock_client.send_email.call_args
         body = kwargs["Content"]["Simple"]["Body"]["Text"]["Data"]
-        self.assertIn("Current website for your organization:", body)
+        self.assertIn("Current websites:", body)
         # spacing should be right between adjacent elements
-        self.assertRegex(body, r"5555\n\nCurrent website for")
+        self.assertRegex(body, r"5555\n\nCurrent websites")
```

Comment:
```suggestion
        self.assertRegex(body, r"5555\n\nCurrent websites:")
```
(Q) Should this be checking everything?